LambdaEngine/Include/Physics/PhysX/QueryFilterCallback.h

```diff
@@ -0,0 +1,44 @@
+#pragma once
+#include "Physics/PhysX/PhysX.h"
+
+namespace LambdaEngine
+{
+	class QueryFilterCallback : public physx::PxQueryFilterCallback
+	{
+	public:
+		DECL_SINGLETON_CLASS(QueryFilterCallback);
+
+		inline virtual physx::PxQueryHitType::Enum preFilter(const physx::PxFilterData& filterData, const physx::PxShape* shape, const physx::PxRigidActor* actor, physx::PxHitFlags& queryFlags) override final
```

Comment:
shape is a pointer -> should be pShape
actor is a pointer -> should be pActor

Reply:
Sure